cpp/ql/lib/semmle/code/cpp/ir/implementation/aliased_ssa/gvn/internal/ValueNumberingInternal.qll

```diff
@@ -152,7 +156,7 @@ private predicate fieldAddressValueNumber(
 ) {
   instr.getEnclosingIRFunction() = irFunc and
   instr.getField() = field and
```

Comment:
I believe this line is redundant now, like the cases above.

Reply:
Oops. You're right.

Reply:
Fixed in 4acae4a2d1b560017ac10ff61043b23e48136eef